modules/apt.sh

```diff
@@ -5,7 +5,7 @@ private_repo_url() {
     local credentials="$2"
     local file="$3"
 
-    echo "https://${credentials}@${repo_url}/ubuntu${file}"
+    echo "https://${credentials}@${repo_url##https://}/ubuntu${file}"
```

Comment:
We should probably make it clear somewhere that we are only supporting the https protocol, we won't be stripping anything else. Maybe add a note to where these URLs are defined.